contexts/language-context.tsx

```
@@ -72,6 +72,15 @@ const translations = {
     frontDeskUpsell: "Front Desk Upsell",
     gestionSolicitudes: "Gestión Solicitudes",
     showingReservations: "Mostrando reservas con fecha de entrada hoy y los próximos 15 días",
+    showingReservationsForDays: "Mostrando reservas para 3 días",
+    searchPlaceholder: "Buscar por localizador o nombre...",
+    roomType: "Tipo de Habitación",
+    frontDeskAgent: "Agente de Recepción",
```

Biome (1.9.4)

[error] This property value named roomType is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named roomType


(lint/suspicious/noDuplicateObjectKeys)